cmd/create/cluster/cmd.go

```diff
@@ -2026,27 +2026,32 @@ func run(cmd *cobra.Command, _ []string) {
 			for _, subnetArg := range subnetIDs {
 				verifiedSubnet := false
 				for _, subnet := range subnets {
-					if awssdk.StringValue(subnet.SubnetId) == subnetArg {
+					if awssdk.ToString(subnet.SubnetId) == subnetArg {
 						verifiedSubnet = true
 					}
 				}
 				if !verifiedSubnet {
+					reg, err := r.AWSClient.GetRegion()
```

Comment:
Changing the `GetRegion` function (left a comment in the client) will save us error handling here.